cluster/manifests/skipper/hpa.yaml

```diff
@@ -21,3 +21,14 @@ spec:
     resource:
       name: memory
       targetAverageUtilization: {{ .ConfigItems.skipper_ingress_target_average_utilization_memory }}
+  behavior:
+    scaleDown:
+      stabilizationWindowSeconds: 600
+      policies:
+      - type: Pods
+        value: 10
+        periodSeconds: 60
```

Comment:
Shouldn't this be 120 seconds, according to the PR description?

Reply:
did not commit this change :D